docs/extend/extend-cli/cli-setting-up.md

```diff
@@ -5,43 +5,37 @@ Before you follow the development tutorials for creating a Zowe&trade; CLI plug-
+### Clone and build your project
 
-## Branches
+1. Create a local development folder named `zowe-tutorial` to clone and build all projects in this folder.
 
-There are two branches in the repository that correspond to different Zowe CLI versions. You can develop two branches of your plug-in so that users can install your plug-in into `@latest` or `@zowe-v2-lts` CLI. Developing for both versions will let you take advantage of new core features quickly and expose your plug-in to a wider range of users.
+2. Clone [`zowe-cli-sample-plugin`](https://github.com/zowe/zowe-cli-sample-plugin/#zowe-cli-sample-plug-in) and build from source.
 
-The `master` branch of Sample Plug-in is compatible with the `@zowe-v2-lts` version of core CLI (Zowe LTS release).
+    Clone the repository into your development folder to match the following structure:
+    ```
+    zowe-tutorial
+    └── zowe-cli-sample-plugin
+    ```
 
-The `master` branch of Sample Plug-in is also compatible with the `@latest` version of core CLI (Zowe Active Development release) at this time.
+    1. Open a terminal and enter `cd zowe-tutorial` to change directory into your `zowe-tutorial` folder.
+    2. Enter `git clone https://github.com/zowe/zowe-cli-sample-plugin` to clone the `zowe-cli-sample-plugin` repository.
+    3. Enter `cd zowe-cli-sample-plugin` to change directory into your `zowe-cli-sample-plugin` folder.
+    4. Enter `npm install` to install all dependencies and modules for the project.
+    5. Enter `npm run build` to create a production build.
 
-For more information about the versioning scheme, see [Maintainer Versioning](https://github.com/zowe/zowe-cli/blob/master/docs/MaintainerVersioning.md) in the Zowe CLI repository.
+### Optional step: Run automated tests
 
-### Clone zowe-cli-sample-plugin and build from source
-Clone the repository into your development folder to match the following structure:
-```
-zowe-tutorial
-└── zowe-cli-sample-plugin
-```
-**Follow these steps:**
+We recommend running automated tests on all code changes.
 
-1. `cd` to your `zowe-tutorial` folder.
-2. `git clone https://github.com/zowe/zowe-cli-sample-plugin`
-3. `cd` to your `zowe-cli-sample-plugin` folder.
-4. `git checkout master`
-5. `npm install`
-6. `npm run build`
+To run automated tests:
 
-### (Optional) Run the automated tests
-We recommend running automated tests on all code changes. Follow these steps:
-
-1. `cd` to the `__tests__/__resources__/properties` folder.
-2. Copy `example_properties.yaml` to `custom_properties.yaml`.
-3. Edit the properties within `custom_properties.yaml` to contain valid system information for your site.
-4. `cd` to your `zowe-cli-sample-plugin` folder
-5. `npm run test`
+1. Open a terminal and enter `cd __tests__/__resources__/properties` to change directory into your `__tests__/__resources__/properties` folder.
```

Comment:
Can step 1 here be removed? I don't think there's any need to open a terminal and change directory into the `__tests__/__resources__/properties` folder, if the subsequent step that references that folder is done outside of the terminal.